client/src/modules/locations/province/province.js

```diff
@@ -3,93 +3,104 @@ angular.module('bhima.controllers')
 
 ProvinceController.$inject = [
   'LocationService', 'util', 'NotifyService',
+  'ModalService', '$uibModal', 'uiGridConstants', 'LocationService',
 ];
 
-function ProvinceController(locationService, util, Notify) {
+function ProvinceController(locationService, util, Notify,
+  Modal, $uibModal, uiGridConstants, Location) {
+
   const vm = this;
   vm.session = {};
   vm.view = 'default';
   vm.state = {};
```

Comment:
One more thing - Please check that `vm.state` and `vm.default` are still used.  If they are not, please remove them from all the controllers.

Reply:
@jeremielodi is it fixed ?